samples/core/Multitenancy/examplesingle/Program.cs

```diff
@@ -0,0 +1,123 @@
+﻿// See https://aka.ms/new-console-template for more information
+using ExampleSingle;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+
+Console.WriteLine("Single database and table example.");
+
+// grab some types as sample data
+#pragma warning disable CS8601 // Where clause guarantees non-nullable
+var types = AppDomain.CurrentDomain.GetAssemblies()
```

Comment:
Uhh... This seems like it goes over all loaded assemblies and create MultitenantTypes for all types which have have a namespace - that's a whole lot of types :) You'd probably want to narrow that down via some sort of attribute or interface - but even then I'd recommend against this kind of assembly exploration (especially of the entire domain).

If what we're looking for is a full list of tenants (though not sure that's a good idea), that should probably come from user config.